### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by vogelino**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
+Version 2, December 2004
+
+Copyright (C) 2016 Lucas Vogel <contact@vogelino.com>
```
Comment (Q-rec) [inline LICENSE.md]: Da müsste es schon meine name sein

**Review by Q-rec — COMMENTED** [review]
Please add my name

File: LICENSE.md (hunk 1 above)
Comment (vogelino) [reply LICENSE.md]: Ja stimmt sorry

Comment (vogelino) [reply LICENSE.md]: Okay I've done it. Please accept my request now

**Comment by Q-rec** [comment]
Okay merged. thx for contributing

**Pull request merged**